packages/vuetify/src/mixins/menuable/index.ts

```diff
@@ -112,10 +182,17 @@ export default baseMixins.extend<options>().extend({
   }),
 
   computed: {
+    computedRelativeOffset () {
```

Comment:
None of this is reactive, you should calculate it at the same time as `dimensions` is then store it in data instead of a computed property.

Reply:
You're right that's a way better approach :sweat_smile: